src/rekapi.actor.js

```diff
@@ -659,9 +587,9 @@ rekapiModules.push(function (context) {
    */
   Actor.prototype.getKeyframeProperty = function (property, millisecond) {
     var propertyTrack = this._propertyTracks[property];
-    if (propertyTrack) {
-      return _.findWhere(propertyTrack, { millisecond: millisecond });
-    }
+    var index = propertyIndexInTrack(propertyTrack, millisecond);
+    if (index != null)
```

Comment:
Ok.  If the `!=` is appropriate, then please put `/*jshint eqeqeq: false */` on the line above to indicate that it is intentional.